test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
Sorry, I should have clarified: I added two additional tests for `LshwHelperSpec` and `LshwParserSpec` to test the new GPU functionality. These new tests (and corresponding lshw XML files) work as expected.

However, for some reason the lshw XML output from all of our new GPU servers will fail all of the existing collins tests. I validated this on the current `master` branch and `add_gpu_support` branch. 

I've been poking around with this, but there appears to be something 'funky' with our hardware, not the recent changes (or the tests).